### DISCUSSION TITLE
error: unable to recognize "./overlays/docker-desktop": no matches for kind "KafkaConnect" in version "kafka.strimzi.io/v1beta1" make: *** [Makefile:18: install-components] Error 1

### DISCUSSION THREAD

**Discussion by zhy949kk** [strimzi/strimzi-kafka-operator#8795] [Q&A]
[trace-grapher.zip](https://github.com/strimzi/strimzi-kafka-operator/files/11974669/trace-grapher.zip)
cd trace-grapher
docker-compose run stack-builder
cd deploy-trace-grapher
make install-components

When I run the make install-components , I get an error, and after three days I still haven't found a solution. I'm begging for answers from you guys.

**Comment by scholzj** [comment]
Sorry, but what exactly are you trying to do? What is `trace-grapher`? How does it relate to Strimzi? I'm not sure I really follow what are you doing. Can you add link to some sources, logs etc.? 

One hint based on the title - `KafkaConnect` in version `kafka.strimzi.io/v1beta1` is not supported for a long time. Strimzi migrated to `kafka.strimzi.io/v1beta2` somewhere in 0.23 release. So that might be related?
